lib/issue_exporter/import.rb

```diff
@@ -0,0 +1,87 @@
+require 'net/http'
+require 'json'
+
+module IssueExporting
+  class Importer
+    def initialize(files, owner, repo, token)
+      @files = files
+      @owner = owner
+      @repo = repo
+      @token = token
+    end
+
+    def import
+      @files.each do |file|
+        file_contents = read_file file
+        import_json(file_contents) unless file_contents.nil?
+      end
+    end
+
+    private
+    def import_json(json_obj)
+      if json_obj.class == Hash
+        import_json_hash json_obj
+      elsif json_obj.class == Array
+        import_json_array json_obj
+      else
+        puts "Unknown object: #{json_obj.class}"
+      end
+    end
+
+    def import_json_array(json_array)
+      json_array.each do |json_hash|
+        import_json_hash json_hash
+      end
+    end
+
+    def import_json_hash(json_hash)
+      create_issue(filter_json json_hash)
+    end
+
+    def filter_json(json_hash)
+      json_hash.select { |k,v| allowed_properties.include? k }
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Space missing after comma.